test/data/iam_invalid.yaml

```diff
@@ -0,0 +1,6 @@
+policies:
```

Comment:
This should already be covered in unit test (if not, we should do that in unit test)

Reply:
We do have them in unit test, will remove.